app/linux/src/LightingManager.cpp

```diff
@@ -45,16 +49,36 @@ CHIP_ERROR LightingManager::Init()
 
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
+
     ChipLogProgress(AppServer, "Using GPIO %s", envGPIO);
 
     gpio = std::stoi(envGPIO);
 
-    wiringPiSetupGpio();
-    pinMode(gpio, OUTPUT);
-    
+    struct gpiod_chip *chip;
+    chip = gpiod_chip_open(GPIO_CHIP);
+    if (!chip)
+    {
+        ChipLogError(AppServer, "Failed to open gpiochip: GPIO_CHIP");
```

Comment:
The GPIO chip device isn't printed but instead always `GPIO_CHIP`. A find+replace-all error?

Reply:
Fixed! Thank you!